src/App.js

```diff
@@ -1,24 +1,18 @@
 import React from 'react';
 import './App.css';
-
 import todos from './api/todos';
 import users from './api/users';
+import { TodoList } from './components/TodoList/TodoList';
 
-function App() {
-  return (
-    <div className="App">
-      <h1>Static list of todos</h1>
-      <p>
-        <span>Todos: </span>
-        {todos.length}
-      </p>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
-    </div>
-  );
-}
+const Todos = todos.map(todo => ({
```

Comment:
```suggestion
const preparedTodos = todos.map(todo => ({
```
назви змінних починаються з малої